packages/core/src/components/editable-text/editableText.tsx

```diff
@@ -301,10 +301,15 @@ export class EditableText extends AbstractPureComponent<IEditableTextProps, IEdi
 
     private updateInputDimensions() {
         if (this.valueElement != null) {
-            const { maxLines, minLines, minWidth, multiline } = this.props;
+            const { maxLines, minLines, minWidth, multiline, value } = this.props;
+            const isControlled = value !== undefined && value !== null;
```

Comment:
`value != null` (two chars) is equivalent to _both_ of these checks (null or undefined)